### PR TITLE
move to Oath groupId

### PR DESCRIPTION
**Pull request by johnmcclean**

### CODE REVIEW

File: micro-application-register/build.gradle [hunk 1]
```diff
@@ -16,7 +16,7 @@ modifyPom {
 		url 'https://github.com/aol/micro-server'
```
Comment (morrowgi) [inline micro-application-register/build.gradle]: Should we change the github repo as well, or leave it in /aol/micro-server ?

File: micro-guava/src/test/java/com/oath/micro/server/testing/RestAgent.java [hunk 2]
```diff
@@ -8,9 +8,26 @@
...
+import org.junit.Test;
+
+import static cyclops.data.Range.range;
+import static cyclops.matching.Api.Case;
+import static cyclops.matching.Api.Match;
+import static cyclops.matching.Api.MatchType;
 
 public class RestAgent {
+	@Test
+	public void diet(){
```
Comment (morrowgi) [inline micro-guava/src/test/java/com/oath/micro/server/testing/RestAgent.java]: Think this was mistakenly checked in?

File: micro-application-register/build.gradle (hunk 1 above)
Comment (johnmcclean) [reply micro-application-register/build.gradle]: There is no Oath OSS github repo yet.

File: micro-guava/src/test/java/com/oath/micro/server/testing/RestAgent.java (hunk 2 above)
Comment (johnmcclean) [reply micro-guava/src/test/java/com/oath/micro/server/testing/RestAgent.java]: Yes.

**Comment by johnmcclean** [comment]
Eek -this is the wrong branch!

**Pull request closed without merging**